src/jvm/test/resources/expected_logs/switch_as_first_method_event.txt

```diff
@@ -1,6 +1,8 @@
 = Invalid execution results =
-Parallel part:
+| Thread 1            | Thread 2            |
+| ----------------------------------------- |
 | incTwiceAndGet(): 3 | incTwiceAndGet(): 4 |
+| ----------------------------------------- |
 
 = The following interleaving leads to the error =
```

Comment:
Let's remove "="-s and add ":" to the end of the title (the same for detailed interleavings)

Reply:
Done